### DISCUSSION TITLE
[Feature] Select photos by year and month

### DISCUSSION THREAD

**Discussion by maartenvsco** [immich-app/immich#14258] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love to be able to click on the year and select all photos of that year so I can then delete them from my phone but keep them on the server.
Same goes for selecting by month.

At the moment I have to select the photos of each day and delete them from the phone. This is a lot of work after you just uploaded all your photos from now till 10 year ago.

(Or have the ability to select a range of photos by selecting the first and then the last photo)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
> Or have the ability to select a range of photos by selecting the first and then the last photo

You can hold to select an asset, then drag and keep holding to select a range. It's not the fastest though.

**Comment by bo0tzz** [comment]
#1661
